discovery_engine/test/feed_manager_test.dart

```diff
@@ -247,25 +247,26 @@ Future<void> main() async {
     });
 
     test('removeExcludedSource', () async {
-      final response = await mgr.removeExcludedSource(Source('www.bbc.com'));
+      final response =
+          await mgr.removeExcludedSource(Source('www.example.com'));
       expect(response, isA<ClientEventSucceeded>());
 
       final content = await mgr.getExcludedSourcesList();
       expect(content, isA<ExcludedSourcesListRequestSucceeded>());
       expect(
         (content as ExcludedSourcesListRequestSucceeded).excludedSources,
-        equals({Source('www.nytimes.com')}),
+        equals({Source('www.example.org')}),
       );
     });
 
     test('getExcludedSourcesList', () async {
       await sourceBox.clear();
 
       final excludedSoures = {
-        Source('theguardian.com'),
-        Source('bbc.co.uk'),
-        Source('wsj.com'),
-        Source('www.nytimes.com'),
+        Source('example.com'),
+        Source('example.org'),
+        Source('example.net'),
+        Source('www.example.info'),
```

Comment:
:wrench: This is a valid domain. a part from `example.com`, `example.org`, `example.net`, other domain must in the form `*.example`, unless a standard is saying that they should be used only for documentation/testing pupose.

Reply:
good point. i'll also change other occurences like example.de